src/styles/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:nth-child(-n+4) {
+      margin-right: 4px;
+    }
+  }
+}
+
+
+
+
+
```

Comment:
Only one empty line should be at the end of the file, fix it everywhere
<img width="370" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/3b48d44a-6028-4574-8275-c1763e9da9ca">